src/Directives.jl

```diff
@@ -95,11 +99,20 @@ function parsebracket end
 parsebracket(text :: AbstractString) = parsebracket(getdirective(text)...)
 parsebracket{D}(:: Directive{D}, text) = error("Unknown directive: '$D'")
 
+function parsebracket{D}(TD :: Directive{D}, arg, text)
+    if strip(arg) == ""
+        parsebracket(TD, text)
+    else
+        error("parsebracket not defined for directive '$D' with positional argument.")
+    end
+end
+
 function parsebracket(:: directive"docs", text)
+    eval(Expr(:toplevel, Expr(:using, :Lexicon)))
```

Comment:
I think you probably want to build src/README.md to ../README.md (i.e. the package toplevel).

The think I dislike about `@{}` in docstrings is that it doesn't fail gracefully. i.e. without Docile installed what's the docstring going to look like rendered. This wasn't a problem with building documentation (you need Docile/Lexicon to build), but is with docstrings (which should work without Docile). :s